src/DomCrawler/Form.php

```diff
@@ -211,10 +211,7 @@ public function set(FormField $field): void
         $this->setValue($field->getName(), $field->getValue());
     }
 
-    /**
-     * @return FormField|FormField[]|FormField[][]
-     */
-    public function get($name): FormField|array
```

Comment:
This looks like a BC break. Can't this method return an array anymore?

Reply:
It seems that it can't, but I'm not fully confident about this one. I'll investigate a bit more.

Reply:
Okay, I checked.

Base class (from `symfony/dom-crawler`) can return an array, this one can't as the private method `getFormField()` is already typed `FormField`.

It could be a BC break via inheritance of this class, but as it's `final`, it seems ok to me. What do you think ?

Reply:
LGTM